public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx

```diff
@@ -90,78 +230,127 @@ export const dBHomeComponent = (
-                        })
-                    }
+            const { domainInfo, physicalMeasurementsInfo, surveyInfo, searchWord, popUp } = this.state;
+            return <React.Fragment>
+                <style>{css}</style>
+                <h1 style={{ ...globalStyles.primaryDisplay, ...styles.dBTitle }}>Data Browser</h1>
+                <p style={{ ...styles.dBDesc, ...globalStyles.bodyLead }}>
+                    The Data Browser provides interactive views of the publicly available<i>All of Us </i>
+                    Research Program participant data. Currently, participant provided information, including surveys and physical
+                    measurements taken at the time of participant enrollment, as well as electronic health record data (EHR) are available.
+                EHR data are reported by health care providers and are not participant reported. The <i>All of Us </i>
+                    Research Program data will include more data types over time.<br></br><br></br>
+                    In order to protect participant privacy, the data are de-identified, limited to aggregate counts rounded up to counts of
+                20, and summary demographic information. For more information, please visit our FAQ page.<br></br><br></br>
+                    Please read the public data use statement available below for additional information about our unique dataset and how to
+                acknowledge the <i>All of Us</i> Research Program in any presentations or publications.<br></br><br></br>
+
+                    <button onClick={() => this.closePopUp()} className='disclaimer-btn'>public data use statement</button>
+                </p>
+                <div style={styles.searchIconLayout}>
+                    <div>
+                        <SearchComponent value={searchWord} onChange={(val) => {
+                            this.handleChange(val);
+                        }}
+                            onClear={() => { this.handleChange(''); }} />
+                        <CdrVersionReactComponent />
+                    </div>
+                    <div style={styles.iconlinks}>
+                        <div className='icons' onClick={() => this.iconClickEvent('FAQ')}>
+                            <a href={environment.researchAllOfUsUrl + '/frequently-asked-questions/#data-browser-faqs'}>
+                                <img alt='FAQs'
+                                    src='/assets/icons/icons_faqs.png' />
+                                <span className='icon-link'>FAQs</span>
+                            </a>
+                        </div>
+                        <div className='icons' onClick={() => this.iconClickEvent('Intro-Videos')}>
+                            <a>
+                                <img alt='Introductory Videos'
+                                    src='/assets/icons/icons_introductoryvideo.png' /><span className='icon-link'>Introductory Videos</span>
```

Comment:
Nit: move to new line:
```suggestion
                                <img alt='Introductory Videos'
                                    src='/assets/icons/icons_introductoryvideo.png' />
                                <span className='icon-link'>Introductory Videos</span>
```